internal/venice-common/src/main/java/com/linkedin/venice/helix/HelixExternalViewRepository.java

```diff
@@ -47,42 +49,7 @@ public HelixExternalViewRepository(SafeHelixManager manager) {
   }
 
   public List<ReplicaState> getReplicaStates(String kafkaTopic, int partitionId) {
-    Partition partition = resourceAssignment.getPartition(kafkaTopic, partitionId);
```

Comment:
not in the scope of this PR: This change looks good for this cleanup PR but I'd suggest we move the API to CV based repo only and force MetaDataHandler to instantiate HelixCVRepo instead general repo.

Reply:
SG, done.